frontend/src/pages/NewAgreement.tsx

```diff
@@ -64,6 +65,34 @@ const NewAgreement: FC = () => {
     setStep(step + 1);
   }
 
+  const [openDialogState, setOpenDialogState] = useState(false);
+  const [
+    createCategoryMutationFN,
+    { data: createCategoryData, loading: newCatLoading, error: newCatError },
+  ] = useMutation(addCategoryMutation);
```

Comment:
We would probably want to refetch categories after this mutation, see [group.tsx:62](https://github.com/consenz-io/webapp/blob/dev/frontend/src/contexts/group.tsx#L62) for reference